packages/studio/tailwind.config.ts

```diff
@@ -47,11 +48,20 @@ const generateSafelist = (theme: StudioTailwindTheme): string[] => {
   ];
 };
 
+const transformedUserContent = (userTailwindConfig?.content ?? []).map(
+  (filepath) => {
+    if (path.isAbsolute(filepath)) {
+      return filepath;
+    }
+    return path.join(rootDir, filepath);
+  }
+);
+
 export default {
   content: [
     path.resolve(__dirname, "src/**/*.{ts,tsx}"),
     path.resolve(__dirname, "index.html"),
-    path.resolve(rootDir, "src/**/*.{ts,tsx}"),
+    ...transformedUserContent,
```

Comment:
can the custom Tailwind Playwright test be updated so it has an example of user-specified content?

Reply:
if the user content doesn't get passed in then none of the styles except safelisted styles would show up

Reply:
this is no longer the case now that we have a default, right?

Reply:
the tailwind test doesn't use the default

Reply:
I updated the tailwind test to have specify and use tailwind classes from files outside the default content
the box shadow classes are pretty weird and I don't expect them to be used by the studio/src files